src/System.IO.Pipelines/tests/Configurations.props

```diff
@@ -3,6 +3,8 @@
   <PropertyGroup>
     <BuildConfigurations>
       netcoreapp;
+      netfx;
```

Comment:
NIT: could you make both the netfx and the uap configurations on the tests to be Windows-specific? Even though these two verticals should never be built on non-Windows machines we try to keep all uap and netfx configurations for tests Windows specific. (`netfx-Windows_NT` and `uap-Windows_NT`)